src/modularize.js

```diff
@@ -98,7 +98,7 @@ var isWW = {{{ nodeWWDetection() }}};
 #endif
 
 #if AUDIO_WORKLET
-isWW ||= typeof AudioWorkletGlobalScope !== 'undefined';
```

Comment:
This one might not work since I believe its outside of the module scope where `ENVIRONMENT_IS_AUDIO_WORKLET` is defined.